static/src/javascripts/projects/commercial/modules/dfp/dfp-env.js

```diff
@@ -18,7 +18,7 @@ define([
         prebidEnabled: config.page.edition == 'US' && !('tests' in config && config.tests.commercialHbSonobi),
 
         /* sonobiEnabled: boolean. Set to true if sonobi real-time-bidding is enabled*/
-        sonobiEnabled: 'tests' in config && config.tests.commercialHbSonobi && !config.page.hasPageSkin,
+        sonobiEnabled: 'tests' in config && config.tests.commercialHbSonobi && (!config.page.hasPageSkin || !config.page.isProd),
```

Comment:
I guess I was just weakening the previous clause. I'll go with your suggestion, Regis de Morgan!